### DISCUSSION TITLE
blog/2024/05-enable-giscuss/

### DISCUSSION THREAD

**Discussion by giscus** [csamaf/csamaf.github.io#8] [Blog comments]
# blog/2024/05-enable-giscuss/

Basically, follow the instructions on giscus.app: enable github.com/apps/giscus for the repo hosting our site, enable discussions for the repo hosting our site, create a new discussion category- Ca...

https://csamaf.github.io/blog/2024/05-enable-giscuss/

**Comment by ftschindler** [comment]
Is this working?

**Reply by renefritze** [reply]
Yeah!

**Reply by renefritze** [reply]
How about the other way round?

**Reply by renefritze** [reply]
That works too!
